src/templates/no_result.html

```
@@ -0,0 +1,147 @@
+{% extends "main.html" %}
+{% block title %}
+<title>API LOGS</title>
+{% endblock %}
+
+{% block content %}
+</div>
```

HTMLHint (1.5.0)

[error] Tag must be paired, no start tag: [ </div> ]

(tag-pair)